dev/az/helmfile.yaml

```diff
@@ -0,0 +1,50 @@
+# single env example file
+# helm file dependencies
+# helm plugin install https://github.com/databus23/helm-diff
+# repositories to be installed
+repositories:
+  - name: datarepo-helm
+    url: https://broadinstitute.github.io/datarepo-helm
+
+# helm releases to be deployed
+releases:
+  - name: az-jade-create-secret-manager-secret    # release name
+    namespace: az   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/create-secret-manager-secret   # chart name
+    version: 0.0.6   #    Chart version
+    missingFileHandler: Warn
+    values:
+      - create-secret-manager-secret.yaml      # Value files passed via --values
+  - name: az-jade-gcloud-sqlproxy    # release name
+    namespace: az   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/gcloud-sqlproxy   # chart name
+    version: 0.19.7 #    Chart version
+    missingFileHandler: Warn
+    values:
+      - gcloud-sqlproxy.yaml    # Value files passed via --values
+  - name: az-jade-datarepo-api   # name of this release
+    namespace: az   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/datarepo-api   # the chart name
+    version: 0.0.25    # chart version
+    missingFileHandler: Warn
+    values:
+      - datarepo-api.yaml   # Value files passed via --values
+  - name: az-jade-datarepo-ui   # name of this release
+    namespace: az   # target namespace
+    createNamespace: true
+    chart: datarepo-helm/datarepo-ui   # the chart name
+    version: 0.0.14    # chart version
```

Comment:
I would recommend removing the version here and for the api so that when you run helmfile apply, it will automatically use the latest version

Otherwise, these versions are behind: https://github.com/broadinstitute/datarepo-helm/blob/master/charts/datarepo/Chart.lock